src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/steps/CommonSteps.kt

```diff
@@ -2,78 +2,47 @@ package fi.aalto.cs.apluscourses.e2e.steps
 
 import com.intellij.remoterobot.RemoteRobot
 import com.intellij.remoterobot.stepsProcessing.step
+import com.intellij.remoterobot.utils.attempt
 import fi.aalto.cs.apluscourses.e2e.fixtures.customComboBox
 import fi.aalto.cs.apluscourses.e2e.fixtures.dialog
 import fi.aalto.cs.apluscourses.e2e.fixtures.heavyWeightWindow
 import fi.aalto.cs.apluscourses.e2e.fixtures.welcomeFrame
-import okhttp3.OkHttpClient
-import okhttp3.Request
-import java.io.File
 import java.time.Duration
 
 class CommonSteps(val remoteRobot: RemoteRobot) {
 
-  fun createProject() {
-    with (remoteRobot) {
-      step("Create New Empty Project") {
-        with(welcomeFrame()) {
-          newProjectButton().click()
-          with(dialog("New Project")) {
-            findText("Empty Project").click()
-            button("Next").click()
-            button("Finish").click()
-          }
-        }
-      }
-      step("Add OpenJDK 11") {
-        with(dialog("Project Structure", Duration.ofSeconds(20))) {
-          sidePanel().findText("Project").click()
-          customComboBox("\u001BProject SDK:").dropdown()
-          with(heavyWeightWindow()) {
-            findText("Add SDK").click()
-            HierarchyDownloader.catchHierarchy {
-              heavyWeightWindow().findText("Download JDK...").click()
+    fun createProject() {
```

Comment:
Do I have to reindent my code?

Reply:
this is something I'm not sure about actually... my indentation happens to be identical to the author's. perhaps this could be solved with a common checkstyle https://github.com/Aalto-LeTech/intellij-plugin/issues/570?